### DISCUSSION TITLE
[Feature] Hope immich supports multiple URLs

### DISCUSSION THREAD

**Discussion by zhz-we** [immich-app/immich#13920] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Hello, I hope Immich can support multiple URLs, and then immich automatically selects the fastest url. 

Sometimes after going home, the speed of synchronizing files using the intranet ip (192.168.31.100:2283) is definitely faster than using the domain name. Through domain name synchronization.

the upload speed is limited by the operator, only 5Mb/s

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by mmomjian** [comment]
Thank you, this is a duplicate of #308 , we are unlikely to add this. You can setup hairpin NAT or split DNS to fix this.
